source/partials/_doc_item_example.slim

```diff
@@ -2,20 +2,14 @@ h5 Example
 
 - item.example.each do |example|
 
-  - if example.description.present?
-    p
-      = example.description
-
   - if example.type == "scss"
     figure
       figcaption SCSS
-      pre
```

Comment:
The  `pre` is already being output by the ` code()` method. It was resulting in multiple `pre`s.

here's a copy of some code from the live site.

```html
<figure>
  <figcaption>SCSS</figcaption>
  <pre>
    <pre class="syntax-highlight scss">
      <code>
        $neat-grid:(
          columns: 12,
          gutter: 20px,
        );
      </code>
    </pre>
  </pre>
</figure>
```